### PR TITLE
TE-10379 Added PHP8.1 to CI

### PR DESCRIPTION
**Pull request by dmytro-dymarchuk**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - php: '7.4'
+          - php: ['7.4', '8.1']
```
Comment (dereuromark) [inline .github/workflows/ci.yml]: Invalid syntax it seems

File: architecture-baseline.json [hunk 2]
```diff
@@ -0,0 +1 @@
+[]
```
Comment (dereuromark) [inline architecture-baseline.json:1]: Did u run this from suite nonsplit?

Comment (dmytro-dymarchuk) [reply architecture-baseline.json:1]: No, this file is just needed for Spryker CI.

Comment (dereuromark) [reply architecture-baseline.json:1]: Sure, it is needed for CI, but it should be generated either as valid one or not at all.
This creates the false sense of "all green".
Best to quickly run this from nonsplit by adding the module tmp.

In the future we also need to find a way to run the arch sniffer standalone.

Comment (dmytro-dymarchuk) [reply architecture-baseline.json:1]: `console code:sniff:architectur -u -m SprykerEco.AvalaraTax` creates the same.

**Pull request merged**